src/main/resources/templates/db_scripts/migration_scripts/add_last_updated_date_to_job.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE diku_mod_data_export.job_executions
+SET jsonb = jsonb_set(jsonb, '{lastUpdatedDate}', to_jsonb(now()), true)
```

Comment:
I think it's better to update the lastUpdated date to the completedDate, instead of now, as these are existing jobs

Reply:
@KVupp I thought about it, but not every job has this field, for example, new doesn`t have it, and if we will have job which is stuck, it will not have this date too. Do you think it is an issue if we will add the current date for all jobs?

Reply:
I agree with Andry completedDate doesn't cover all cases

Reply:
That's a good point , my thought was it might create a confusion as to why the dates are different later if someone is looking at the records. I think we can use the completedDate if present, and if not use current date?

Reply:
ok, I will modify the script, verify, and then push changes